Classes/Hook/ProcessDatamap/ProductInheritanceProcessDatamap.php

```diff
@@ -164,95 +179,119 @@ public function processDatamap_afterAllOperations(DataHandler $dataHandler): voi
      *
      * @param $identifier
      */
+
+    // phpcs:disable Generic.Metrics.CyclomaticComplexity
     protected function processProductRecordOverlays($identifier): void
     {
+        $inheritMode = false;
         $productRow = $this->productDatamap[$identifier];
 
+        // $productRow isn't an arrat if parent product was saved and then
+        // called function to save children, then inheritmode = true
         if (!is_array($productRow)) {
-            $productRow = BackendUtility::getRecord(
-                ProductRepository::TABLE_NAME,
-                $identifier,
-                'parent,product_type'
-            );
```

Comment:
yay! Probably a very good thing to get rid of this call!

Reply:
Well, it is still there, but in another place.